packages/babel-helper-builder-react-jsx-experimental/src/index.js

```diff
@@ -43,16 +66,16 @@ export function helper(babel, options) {
         }
       }
 
-      const source = t.jsxAttribute(
-        t.jsxIdentifier("__source"),
-        t.jsxExpressionContainer(makeSource(path, state)),
-      );
       const self = t.jsxAttribute(
         t.jsxIdentifier("__self"),
         t.jsxExpressionContainer(t.thisExpression()),
       );
+      const source = t.jsxAttribute(
+        t.jsxIdentifier("__source"),
+        t.jsxExpressionContainer(makeSource(path, state)),
+      );
 
-      path.pushContainer("attributes", [source, self]);
+      path.pushContainer("attributes", [self, source]);
```

Comment:
it doesn't matter but this matches the order created in the old transform so I've reordered things here to avoid updating a lot of fixtures